addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -310,6 +310,19 @@ export class Rtc extends Record {
                 this.sfuClient?.disconnect();
             }
         });
+        browser.navigator.serviceWorker?.addEventListener("message", ({ data: { action, id } }) => {
+            if (action === "join_call") {
+                const channel = this.store.Thread.get({ model: "discuss.channel", id });
+                if (!channel) {
+                    return;
+                }
```

Comment:
Why ignore in this case? If the button was presented to the user but they didn't know the channel yet, it's probably better to call `getOrFetch` instead of discarding.